build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```diff
@@ -275,6 +275,10 @@ extraJavaModuleInfo {
         exportAllPackages()
         requires("java.compiler")
     }
+    module("pl.tkowalcz.tjahzi:log4j2-appender-nodep", "loki.log4j2") {
```

Comment:
@mxtartaglia-sl @jasperpotts Do we really need to bring in yet another dependency chain? 

Why not just log to a JSON layout and have OTel or Grafana Agent stream to Loki from the log file?

Reply:
Thanks for the suggestion! It makes sense, especially after what Jasper communicated.
I added a promtail agent to scrap the logs from the filesystem instead.

I understand that adding the dependency here allows anybody to reference this dependency in the rest of the code, which is not desired. Still, I also think (I don't know how often the case is) that adding dependencies for an example project should not be as restrictive as it is for things running elsewere; maybe those can have dependency files?  

Anyway it is solved now